src/components/widgets/FileWidget.js

```diff
@@ -4,6 +4,7 @@ import PropTypes from "prop-types";
 import { dataURItoBlob, shouldRender, setState } from "../../utils";
 
 function addNameToDataURL(dataURL, name) {
+  name=encodeURIComponent(name)
```

Comment:
you've got a lint error here.

Do this instead:
```
  function addNameToDataURL(dataURL, name) {
    return dataURL.replace(";base64", `;name=${encodeURIComponent(name)};base64`); 
}
```

Reply:
Sorry @graingert, did not see your comment until now.